src/Microsoft.Azure.NotificationHubs/NotificationDetails.cs

```diff
@@ -143,6 +143,27 @@ string NotificationState
         [DataMember(Name = ManagementStrings.GcmOutcomeCounts, IsRequired = false, Order = 1013, EmitDefaultValue = false)]
         public NotificationOutcomeCollection GcmOutcomeCounts { get; set; }
 
+        /// <summary>
+        /// Gets or sets the notification FCM outcome counts.
+        /// </summary>
+        ///
+        /// <remarks>
+        /// This field acts as proxy for GcmCredentials at the moment as FcmNotificaiton instances are converted to GcmNotification when
+        /// sent out to the Notification Hub, making it impossible to distinguish the two in the response as from the NH's perspective, the
+        /// GcmNotification was received. See <see cref="M:Microsoft.Azure.NotificationHubs.NotificationHubClient.SendNotificationImplAsync"/> and
+        /// <see cref="M:Microsoft.Azure.NotificationHubs.NotificationHubClient.SendScheduledNotificationImplAsync"/> methods for conversion details.
+        /// </remarks>
+        /// 
+        /// <value>
+        /// The notification FCM outcome counts.
+        /// </value>
+        [DataMember(Name = ManagementStrings.FcmOutcomeCounts, IsRequired = false, Order = 1016, EmitDefaultValue = false)]
+        public NotificationOutcomeCollection FcmOutcomeCounts
```

Comment:
This field simply forwards the GcmOutcomeCounts. It does this because we convert `FcmNotification` into `GcmNotifications` when we send them to the service (which is not aware of FCM types) and so the service actually returns `GcmOutcomeCounts`.